designs/SearchCommandSpec.md

```diff
@@ -0,0 +1,98 @@
+# Search CLI Tool for NuGet.exe and Dotnet.exe
+
+* Status: **Implementing**
+* Author(s): [Advay Tandon](https://github.com/advay26)
+* Issue: [9704](https://github.com/NuGet/Home/issues/9704) - Search CLI Tool for NuGet.exe and Dotnet.exe
+
+## Problem Background
+
+Search functionality from the command line for NuGet.exe and the .NET Core CLI has been a feature request from the NuGet open source community for some time. Additionally, the NuGet API is already equipped to handle search requests, so an extension of this service to command line users would be both practical to implement and also highly valuable.
+
+## Who are the customers
+
+All NuGet.exe and .NET CLI users.
```

Comment:
Why do we want to support `NuGet.exe` customers? Is `nuget.exe list` not enough? It may be helpful to describe the benefits and differences between `list` and `search`

Reply:
I have attempted to address this in the 'Who are the customers' section. Let me know where I've fallen short in my explanation and I'd be happy to amend it further!